File: 백준/Bronze/11050. 이항 계수 1/이항 계수 1.cc
#include <iostream>

using namespace std;

long factorial(long num) {
    if (num <= 1)
        return 1;
    return num * factorial(num - 1);
}

int main() {
    int n, k;
    cin >> n >> k;

    cout << factorial(n) / (factorial(k) * factorial(n - k));
}

File: 백준/Bronze/11050. 이항 계수 1/이항 계수 1.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {

    static int N, K;

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        StringTokenizer st = new StringTokenizer(br.readLine());
        N = Integer.parseInt(st.nextToken());
        K = Integer.parseInt(st.nextToken());

        int top = factorial(N);
        int bottom = factorial(K) * factorial(N - K);

        System.out.println(top / bottom);
    }

    static int factorial(int num) {
        if (num <= 1) {
            return 1;
        }
        return num * factorial(num - 1);
    }
}